contracts/UFragmentsPolicy.sol

```diff
@@ -65,6 +65,9 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
+    // CPI value at the time of launch, DECIMALS Fixed point number
+    uint256 private baseCpi;
```

Comment:
Conceptually, I'd put this next to deviationThreshold in the code.

Reply:
Done 👍